src/components/node/stanford-page/home-page/today-hours.tsx

```diff
@@ -127,12 +127,12 @@ const TodayLibraryHours = ({branchId}: {branchId?: string}) => {
     )
   }
   const {closedAllDay, isOpen, openingTime, closingTime, afterClose} = libraryHours
-  const hoursDisplay = !closedAllDay && (isOpen ? "Closes at " + closingTime : afterClose ? "Closed at " + closingTime : "Opens at " + openingTime)
+  const hoursDisplay = !closedAllDay && (isOpen ? "Open until " + closingTime : afterClose ? "Closed until " + openingTime : "Closed until " + openingTime)
 
   return (
     <>
       <div
-        className="mb-4 flex justify-between text-black"
+        className="mb-4 flex text-black"
```

Comment:
Do you want `items-center` as well to make sure the elements are centered with each other vertically.